### PR TITLE
Translate to Japanese Chapter0

### PR DESCRIPTION
**Pull request by hiromu166**
This is the first PR about translating to Japanese.

In this PR, I translate chapter0/1.mdx.

### CODE REVIEW

**Comment by hiromu166** [comment]
Hi @lewtun.
I've translated Chapter0/1.mdx into Japanese.
Could you check this PR?

File: .github/workflows/build_documentation.yml [hunk 1]
```diff
@@ -14,6 +14,6 @@ jobs:
       package: course
       path_to_docs: course/chapters/
       additional_args: --not_python_module
-      languages: ar bn en es fa fr he hi ko pt ru th tr zh
+      languages: ar bn en es fa fr he hi ja ko pt ru th tr zh
```
Comment (lewtun) [inline .github/workflows/build_documentation.yml:17]: Great job on adding this to the PR!

**Review by lewtun — APPROVED** [review]
Thank you for this contribution @hiromu166 - this looks perfect 🚀 !!

We can't render the website until `chapter1/1.mdx` is translated, but this looks great as it is and I'll merge once the CI passes

**Comment by hiromu166** [comment]
Thank you @lewtun for your review!
I'll translate `chapter1/1.mdx` soon.

**Comment by lewtun** [comment]
> Thank you @lewtun for your review! I'll translate `chapter1/1.mdx` soon.

Fanstatic, thank you 😍 !

**Pull request merged**